core/src/providers/anthropic.rs

```diff
@@ -209,9 +209,9 @@ impl LLM for AnthropicLLM {
             completions.push(Tokens {
                 // Anthropic only return the text
                 text: response.completion.clone(),
-                tokens: None,
-                logprobs: None,
-                top_logprobs: None,
+                tokens: Some(vec![]),
```

Comment:
@spolu  doing this because [this line](https://github.com/dust-tt/dust/blob/16c59aa7e18518a2fa90876bb2f0b8f37444f3e2/core/src/providers/llm.rs#L254) crashes at unwrap when logprobs is `None`

Reply:
Let's fix the line instead?

Reply:
Will do in a separate PR to avoid dismissing this review